src/main/resources/org/jenkinsci/test/acceptance/selenium/scroller.js

```diff
@@ -1,8 +1,9 @@
 // Visually navigate to the element in order to interact with it.
 
 var eYCoord = arguments[0];
+var eXCoord = arguments[1];
 
 // Scroll to the element. It will appear at the top edge of the screen.
 // We subtract a bit so as to accommodate fixed position banners at the top
 // (e.g. breadcrumbs, tabbars etc), making sure they are not hiding the element.
-window.scrollTo(0, eYCoord - 200);
\ No newline at end of file
+window.scrollTo(eXCoord - 100, eYCoord - 200);
```

Comment:
Did you find any element that makes this - 100 necessary? If yes, probably a clarifying comment would be appropriate.